flow/table.go

```diff
@@ -429,8 +419,6 @@ func (ft *Table) Run() {
 			if ok {
 				ft.reply <- ft.onQuery(query)
 			}
-		case now := <-nowTicker.C:
```

Comment:
Still testing the whole patch but here removing this ticker we trigger an issue as we have to wait an update or an expire timeout to get the ft.state checked. If have these parameters set to something high let say 5 min, we have to wait 5 mins before having to table stopped. So I propose to not remove the ticker.

Reply:
agreed

Reply:
I replaced it by a chan quit, as it make no sense to have a ticker just to exit a loop (evaluate more often)

Reply:
nice, thx